### DISCUSSION TITLE
jetbrains plugin gives weird completion

### DISCUSSION THREAD

**Discussion by dmwyatt** [community/community#9863] [Copilot Conversations] [answered]
![image](https://i.imgur.com/hpx9v8L.gif))

In PyCharm, I often delete the newline between lines and then press Enter to clean up indentation automatically.  Copilot always makes weird suggestions like the one illustrated above.  This time I didn't capture one of the multi-line ones it does where the characters that are supposed to be there (e.g. ['r', 'e', 't', 'u', 'r', 'n', ' ', 'F', 'a', 'l', 's', 'e']) are interspersed across multiple lines.

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! 
Could you add the version of the GitHub Copilot plugin you're using, please?

This is a bug and we'll get it fixed (a similar bug was fixed in a recent update).

**Reply by dmwyatt** [reply]
Now I'm on 1.1.4.220.  I don't know what the version was at the time of my original post.  In fact, right now I can't seem to reproduce this behavior.

**Reply by dmwyatt** [reply]
![pycharm64_rPT1gfXTcO](https://user-images.githubusercontent.com/163359/150577089-fc1485ab-d258-4f22-8f42-dc66b77b8ca1.gif)

(Not exactly clear in that animation, but I'm pressing the hotkey for the Jetbrains `Delete Line` function a couple of times)

I reproduced it in a different place.  Still on 1.1.4.220.

**Reply by jansorg** [reply]
Thanks @dmwyatt! This is clearly a bug, we'll investigate and fix.

**Reply by jansorg** [reply]
@dmwyatt Are you still seeing this with the latest version (1.1.12)? I think that it's fixed, but your feedback would be great.

**Reply by dmwyatt** [reply]
It seems to be fixed.  I'll report back if I see it again.
